practice/react-practice-one/src/components/ConfirmModal/ConfirmModal.jsx

```diff
@@ -0,0 +1,51 @@
+/* Import dependencies */
+import PropTypes from 'prop-types';
+
+/* Import CSS */
+import './confirmModal.css';
+
+/* Import constants */
+import { MODAL_TITLES } from '../../constants';
+
+import Text from '../Text/Text.jsx';
+import Button from '../Buttons/Button.jsx';
+
+const ConfirmModal = ({ id, title, onSubmit, onClose, isMutating, isOpen }) => {
```

Comment:
@JennyDoJD why should we need `id` and `isMutating`?

Reply:
how about `onConfirm` instead `onSubmit`?

Reply:
@JennyDoJD did you check my comment?

Reply:
I used `isMutating` because I wanted to determine if the ConfirmModal was mutating but it was a bit redundant so I removed it and used `id` to determine the ID of the product to confirm

Reply:
how about this comment `how about onConfirm instead onSubmit?`

Reply:
Sorry. I missed this comment, so I fixed it.